lmp/bb-build.sh

```diff
@@ -13,11 +13,11 @@ function finish() {
         # we need to check that because it can't be available in old containers
         if command -v xvfb-run >/dev/null 2>&1 ; then
             # producing bootchart.svg
-            run xvfb-run ../layers/openembedded-core/scripts/pybootchartgui/pybootchartgui.py \
+            run xvfb-run $(realpath $BUILDDIR/../layers/openembedded-core/scripts/pybootchartgui/pybootchartgui.py) \
```

Comment:
I don't see a reason for not squashing it to the previous commit.

Reply:
@quaresmajose I would probably do the other way around, have this patch first before moving the content to the finish function and then the other patch on top that basically just moves the same content but to a function.

This way it is easy to compare and isolate the changes.

Reply:
makes sense